### ISSUE TITLE
Accessibility guide timeout

### ISSUE THREAD

**Issue by notklaatu** [patternfly/patternfly#3343]
Could be an issue with Cloudflare but I'm unable to reach the Accessibility Guide. I'm only reporting it because I thought Cloudflare was supposed to prevent these kinds of errors (although I'm no expert on Cloudflare).

```
$ curl --head --location https://pf4.patternfly.org/accessibility-guide                        
HTTP/1.1 522 Origin Connection Time-out
Date: Wed, 29 Jul 2020 20:51:31 GMT
Content-Type: text/html
Connection: keep-alive
Set-Cookie: __cfduid=de4056838be3c4487795e1382f69a56421596055860; expires=Fri, 28-Aug-20 20:51:00 GMT; path=/; domain=.patternfly.org; HttpOnly; SameSite=Lax; Secure
Cache-Control: no-store, no-cache
cf-request-id: 043df0cd2800000a4890838200000001
Expect-CT: max-age=604800, report-uri="https://report-uri.cloudflare.com/cdn-cgi/beacon/expect-ct"
Server: cloudflare
CF-RAY: 5ba9b7284d880a48-KIX
```

**Comment by redallen** [comment]
Surge.sh (our free hosting provider) is currently experiencing an outage. In the meantime you can view the MD file for the page [here.](https://github.com/patternfly/patternfly/blob/master/src/site/pages/accessibility-guide.md)

Sorry for the inconvenience!
